File: benchmarks/numerical/sort_10k.go
package main

import (
	"fmt"
	"math/rand"
	"sort"
	"time"
)

func main() {
	data := make([]int, 10000)
	for i := range data {
		data[i] = rand.Intn(1000000)
	}
	start := time.Now()
	sort.Ints(data)
	fmt.Println(time.Since(start).Microseconds())
}

File: benchmarks/numerical/sort_10k.rs
use std::time::Instant;
fn main() {
    let mut data: Vec<i32> = (0..10000).map(|i| (i * 7919 + 104729) % 1000000).collect();
    let start = Instant::now();
    data.sort();
    println!("{}", start.elapsed().as_micros());
}
